pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -209,7 +209,7 @@ func (p *Provider) createLaunchTemplate(ctx context.Context, options *amifamily.
 			Monitoring: &ec2.LaunchTemplatesMonitoringRequest{
 				Enabled: aws.Bool(options.DetailedMonitoring),
 			},
-			SecurityGroupIds: aws.StringSlice(options.SecurityGroupsIDs),
+			SecurityGroupIds: lo.Map(options.SecurityGroups, func(secGroup *ec2.SecurityGroup, _ int) *string { return secGroup.GroupId }),
```

Comment:
```suggestion
			SecurityGroupIds: lo.Map(options.SecurityGroups, func(s *ec2.SecurityGroup, _ int) *string { return s.GroupId }),
```

nit: shorten the name?